test/unit/rules/self-closing-void-elements-test.js

```diff
@@ -874,5 +938,32 @@ generateRuleTests({
         `);
       },
     },
+    {
+      template: 'foo<wbr data-custom="50" {{my-modifier true "baz"}} as |paramA paramB|>bar',
```

Comment:
Can we make sure that comments and children are also tested?
```hbs
foo<div data-custom="50" {{my-modifier true "baz"}} as |paramA paramB| {{!comment}}>child</div>bar
```

Reply:
Actually it doesn't make sense to test children inside a void element so I just added the comment part instead: https://github.com/ember-template-lint/ember-template-lint/pull/2583#discussion_r952816841